multimedia/gstreamer1/Makefile

```diff
@@ -17,7 +17,7 @@ LICENSE_FILE=	${WRKSRC}/COPYING
 PORTSCOUT=	limitw:1,even
 
 CFLAGS+=	-Wno-format
-USES=		bison gettext gmake gnome libtool pathfix pkgconfig tar:xz
+USES=		bison gettext gmake gnome libtool pathfix pkgconfig tar:xz python
```

Comment:
huh, I wonder what started needing python in this update. I haven't switched the build to meson.

Reply:
Oups missing cleanup :)